firefox-ios/Client/Experiments/RecordedNimbusContext.swift

```diff
@@ -0,0 +1,185 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Glean
+import Shared
+
+import func MozillaAppServices.getCalculatedAttributes
+import func MozillaAppServices.getLocaleTag
+import struct MozillaAppServices.JsonObject
+import protocol MozillaAppServices.RecordedContext
+import MozillaRustComponents
+
+private extension Double? {
+    func toInt64() -> Int64? {
+        return self == nil ? nil : Int64(self!)
```

Comment:
I believe we've enabled the swiftlint `force_cast` rule, so this will likely fail on CI. We could always do something like:

```suggestion
        guard let self = self else { return nil }
        return Int64(self)
```